tgui/packages/tgui/interfaces/ShuttleConsole.js

```diff
@@ -0,0 +1,71 @@
+import { useBackend } from '../backend';
+import { Button, LabeledList, Box, Fragment, Section } from '../components';
+import { Window } from '../layouts';
+import { LabeledListItem } from '../components/LabeledList';
+
+export const ShuttleConsole = (props, context) => {
+  const { act, data } = useBackend(context);
+  // get stuff
+  return (
+    <Window resizable>
+      <Window.Content>
+        <LabeledList>
+          <LabeledList.Item label="Location">
+            {data.status ? (
+              data.status
+            ) : (
+              <Box color="red">
+                *MISSING*
+              </Box>
+            )}
+          </LabeledList.Item>
+          {!!data.shuttle && (// only show this stuff if there's a shuttle
+            <Fragment>
+              {!!data.docking_ports_len && (
+              // I am not sure if I am nesting this stuff right...
+                <Fragment>
+                  <LabeledList.Item label={"Send to "}>
+                    {data.docking_ports.map(port => (
+                      <Button
+                        icon="chevron-right"
+                        key={port.name}
+                        content={port.name}
+                        // TODO: should button have icon?
```

Comment:
If you can find a good icon, sure. Here's the [list](https://fontawesome.com/icons?d=gallery).